packages/notify-client/src/types/client.ts

```diff
@@ -201,14 +210,17 @@ export declare namespace NotifyClientTypes {
 
   interface SubscriptionResponseJWTClaims extends CommonResponseJWTClaims {
     act: "notify_subscription_response";
+    sbs: NotifyServerSubscription[];
```

Comment:
Adding for consistency

```suggestion
    sbs: NotifyServerSubscription[]; // array of [Notify Server Subscriptions]
```